common/stream-model.cpp

```diff
@@ -1257,16 +1263,27 @@ namespace rs2
             ImGui::PushStyleColor(ImGuiCol_ButtonHovered, header_window_bg);
             ImGui::PushStyleColor(ImGuiCol_ButtonActive, header_window_bg);
 
-            float y_offset = 0;
             if (show_stream_details)
             {
                 y_offset += 30;
             }
 
             std::string label = rsutils::string::from() << "IMU Stream Info of " << profile.unique_id();
 
+            int const line_h = 18;
```

Comment:
Is this height true for all font sizes? For window mode and maximized mode?
Note, we have "font size" setting under performance definition. Not sure it affects the stream views.

Reply:
It's the height per line and was fixed before, I just moved it